cmd/compute-domain-controller/daemonset.go

```diff
@@ -173,6 +174,12 @@ func (m *DaemonSetManager) Create(ctx context.Context, namespace string, cd *nva
 		return nil, fmt.Errorf("error creating ResourceClaimTemplate: %w", err)
 	}
 
+	// Get feature gates as map for template rendering
+	featureGatesMap := map[string]bool{}
+	if m.config.featureGateConfig != nil {
+		featureGatesMap = m.config.featureGateConfig.ToMap()
```

Comment:
Add a type with a short, explicit name for `map[string]bool{}`?

E.g.
```golang
type FeatureGates map[string]bool{}

...

	if m.config.featureGateConfig != nil {
		featureGates = m.config.featureGateConfig.Get()



...

	templateData := DaemonSetTemplateData{
		Namespace:                 m.config.driverNamespace,
...
		FeatureGates:              featureGates,
	}
```

Hm. That's probably in conflict with 
```
k8s.io/component-base/featuregate
github.com/NVIDIA/k8s-dra-driver-gpu/pkg/featuregate
```

Reply:
I actually think it's clearer without an intermediate type. The point of this function is specifically to generate a map and the return type reflects that.